docs/source/main-algorithms/core-classes/delta.rst

```diff
@@ -0,0 +1,15 @@
+..
+    Copyright (c) 2025 Joseph Edwards
+
+    Distributed under the terms of the GPL license version 3.
+
+    The full license is in the file LICENSE, distributed with this software.
+
+.. currentmodule:: _libsemigroups_pybind11
+
+The delta function
+==================
+
+This page contains the documentation for the :any:`delta` function. 
```

Comment:
```suggestion
This page contains the documentation for the :py:func:`delta` function. 
```
This is because the file and the function have the same name.